admin/test/test_packaging.py

```diff
@@ -199,7 +199,7 @@ def assert_deb_content(test_case, expected_paths, package_path):
             continue
         actual_paths.add(FilePath('/').descendant(f.segmentsFrom(output_dir)))
 
-    test_case.assertEqual(expected_paths, actual_paths)
+    test_case.assertEqual(set(), expected_paths.difference(actual_paths))
```

Comment:
Yeah, it's to check that the package *contains* the expected_paths  rather than checking that the list of packaged files exactly matches `expected_paths`.

And if any of `expected_paths` are not included in the package they will be included in the assertion failure message making it easier to debug the failure.